logic/transaction.js

```diff
@@ -635,7 +635,15 @@ class Transaction {
 				sender
 			) || false;
 		if (!fee || transaction.fee !== fee) {
-			return setImmediate(cb, 'Invalid transaction fee');
+			err = 'Invalid transaction fee';
+
+			if (exceptions.transactionFee.indexOf(transaction.id) > -1) {
+				this.scope.logger.error(err);
```

Comment:
this should not be an error as it is expected behavior. debug or info fits better.

Reply:
It's still an error, but we just skip it. Maybe it makes sense, but all other exceptions are currently logged like that, changing that is out of scope of that issue.